scala/friesian.serving/README.md

```diff
@@ -1,29 +1,105 @@
-## Friesian gRPC Recommendation Framework
+## Friesian Serving Recommendation Framework
 
+### Architecture of the serving pipelines
+
+The diagram below demonstrates the components of the friesian serving system, which typically consists of three stages:
+
+- Offline: Preprocess the data to get user/item DNN features and user/item Embedding features. Then use the embedding features and embedding model to get embedding vectors.
+- Nearline: Retrieve user/item profiles and keep them in the Key-Value store. Retrieve item embedding vectors and build the faiss index. Make updates to the profiles from time to time.
+- Online: Trigger the recommendation process whenever a user comes. Recall service generate candidates from millions of items based on embeddings and the deep learning model ranks the candidates for the final recommendation results.
+
+![Architecture](./src/main/resources/images/architecture.png "The Architecture of the Friesian serving pipeline")
```

Comment:
This chart is specifically for the demo case; need to update the chart to make it more general